frontend/shared/scss/base/text-styles.scss

```diff
@@ -888,12 +888,11 @@
     @extend .style-illustration-img;
 }
 
-.style-icon-with-progress {
+.style-image-with-icon {
     position: relative;
     margin-right: 5px;
-    color: $color-dark;
 
-    &.complete {
+    &.blue {
```

Comment:
We gebruiken meestal 'primary', zou geen 'blue' gebruiken omdat het zeker niet altijd blauw zal zijn